aot-tests/src/test/resources/com/dylibso/chicory/testing/InterpreterFallbackTest.testSilentInterpreterFallback-indirect.approved.txt

```diff
@@ -0,0 +1,20 @@
+com.dylibso.chicory.testing.InterpreterFallbackTest.lambda$testSilentInterpreterFallback$1
```

Comment:
I need to squint my eyes too much to understand what those approvals are testing and how ...

I'm open to proposals:
- a different way of testing
- more comments around what to check

Reply:
I remember this was clear enough in the early stages of this PR (in comments), but I struggle to see it now, e.g. seems like the approvals are making the check more difficult to spot now.

Reply:
Should I go back to using the big list in comments?  Or is there a better strategy?

Reply:
I trust you :-)